From 69ee22688844e0ca9ecce19335fdd7982429c922 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Tue, 11 Jan 2011 15:42:38 +0800
Subject: [PATCH] ENGR00137844 MX53: update MFG defconfig

- Add MX53 SMD and LOCO board support
- Build in SATA, DA9053 regulator
- Remove unnecessory drivers

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 arch/arm/configs/imx5_updater_defconfig |   65 ++++++++++++++++++++++++++++---
 1 files changed, 59 insertions(+), 6 deletions(-)

diff --git a/arch/arm/configs/imx5_updater_defconfig b/arch/arm/configs/imx5_updater_defconfig
index d5ef907..58d4888 100644
--- a/arch/arm/configs/imx5_updater_defconfig
+++ b/arch/arm/configs/imx5_updater_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.35.3
-# Fri Nov 26 10:35:54 2010
+# Tue Jan 11 21:48:16 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -93,6 +93,7 @@ CONFIG_SIGNALFD=y
 CONFIG_TIMERFD=y
 CONFIG_EVENTFD=y
 CONFIG_SHMEM=y
+# CONFIG_ASHMEM is not set
 CONFIG_AIO=y
 CONFIG_HAVE_PERF_EVENTS=y
 CONFIG_PERF_USE_VMALLOC=y
@@ -249,6 +250,8 @@ CONFIG_MACH_MX51_3DS=y
 CONFIG_MACH_MX51_BABBAGE=y
 CONFIG_MACH_MX53_EVK=y
 CONFIG_MACH_MX53_ARD=y
+CONFIG_MACH_MX53_SMD=y
+CONFIG_MACH_MX53_LOCO=y
 CONFIG_MACH_MX50_ARM2=y
 CONFIG_MACH_MX50_RDP=y
 
@@ -268,6 +271,7 @@ CONFIG_MXC_PWM=y
 CONFIG_ARCH_HAS_RNGC=y
 CONFIG_ARCH_MXC_IOMUX_V3=y
 CONFIG_MXC_DVFS_PER=y
+# CONFIG_MXC_ZQ_CALIBRATION is not set
 
 #
 # Processor Type
@@ -590,7 +594,40 @@ CONFIG_SCSI_LOWLEVEL=y
 # CONFIG_SCSI_DEBUG is not set
 # CONFIG_SCSI_DH is not set
 # CONFIG_SCSI_OSD_INITIATOR is not set
-# CONFIG_ATA is not set
+CONFIG_ATA=y
+# CONFIG_ATA_NONSTANDARD is not set
+CONFIG_ATA_VERBOSE_ERROR=y
+# CONFIG_SATA_PMP is not set
+
+#
+# Controllers with non-SFF native interface
+#
+CONFIG_SATA_AHCI_PLATFORM=y
+CONFIG_ATA_SFF=y
+
+#
+# SFF controllers with custom DMA interface
+#
+CONFIG_ATA_BMDMA=y
+
+#
+# SATA SFF controllers with BMDMA
+#
+# CONFIG_SATA_MV is not set
+
+#
+# PATA SFF controllers with BMDMA
+#
+# CONFIG_PATA_FSL is not set
+
+#
+# PIO-only SFF controllers
+#
+# CONFIG_PATA_PLATFORM is not set
+
+#
+# Generic fallback / legacy drivers
+#
 # CONFIG_MD is not set
 # CONFIG_PHONE is not set
 
@@ -633,6 +670,7 @@ CONFIG_INPUT_KEYBOARD=y
 # CONFIG_KEYBOARD_SUNKBD is not set
 # CONFIG_KEYBOARD_XTKBD is not set
 CONFIG_KEYBOARD_MXC=y
+# CONFIG_KEYBOARD_MPR121 is not set
 # CONFIG_INPUT_MOUSE is not set
 # CONFIG_INPUT_JOYSTICK is not set
 # CONFIG_INPUT_TABLET is not set
@@ -662,6 +700,7 @@ CONFIG_TOUCHSCREEN_MXC=y
 # CONFIG_TOUCHSCREEN_TSC2007 is not set
 # CONFIG_TOUCHSCREEN_W90X900 is not set
 # CONFIG_TOUCHSCREEN_TPS6507X is not set
+# CONFIG_TOUCHSCREEN_DA9052 is not set
 CONFIG_INPUT_MISC=y
 # CONFIG_INPUT_AD714X is not set
 # CONFIG_INPUT_ATI_REMOTE is not set
@@ -690,10 +729,10 @@ CONFIG_HW_CONSOLE=y
 # CONFIG_VT_HW_CONSOLE_BINDING is not set
 CONFIG_DEVKMEM=y
 # CONFIG_SERIAL_NONSTANDARD is not set
-CONFIG_FM_SI4702=y
+# CONFIG_FM_SI4702 is not set
 CONFIG_MXC_IIM=y
 CONFIG_MXS_VIIM=y
-CONFIG_IMX_SIM=y
+# CONFIG_IMX_SIM is not set
 
 #
 # Serial drivers
@@ -819,6 +858,7 @@ CONFIG_GPIOLIB=y
 #
 # MODULbus GPIO expanders:
 #
+# CONFIG_DA9052_GPIO_ENABLE is not set
 # CONFIG_W1 is not set
 CONFIG_POWER_SUPPLY=y
 # CONFIG_POWER_SUPPLY_DEBUG is not set
@@ -828,6 +868,7 @@ CONFIG_APM_POWER=y
 # CONFIG_BATTERY_DS2760 is not set
 # CONFIG_BATTERY_DS2782 is not set
 # CONFIG_BATTERY_BQ27x00 is not set
+# CONFIG_BATTERY_DA9052 is not set
 # CONFIG_BATTERY_MAX17040 is not set
 # CONFIG_BATTERY_DS2438 is not set
 CONFIG_HWMON=y
@@ -852,6 +893,7 @@ CONFIG_HWMON=y
 # CONFIG_SENSORS_ADT7475 is not set
 # CONFIG_SENSORS_ASC7621 is not set
 # CONFIG_SENSORS_ATXP1 is not set
+# CONFIG_SENSORS_DA9052 is not set
 # CONFIG_SENSORS_DS1621 is not set
 # CONFIG_SENSORS_F71805F is not set
 # CONFIG_SENSORS_F71882FG is not set
@@ -907,7 +949,9 @@ CONFIG_HWMON=y
 # CONFIG_SENSORS_W83627EHF is not set
 # CONFIG_SENSORS_LIS3_SPI is not set
 # CONFIG_SENSORS_LIS3_I2C is not set
-CONFIG_SENSORS_ISL29003=y
+# CONFIG_SENSORS_ISL29003 is not set
+# CONFIG_MXC_MMA8450 is not set
+# CONFIG_MXC_MMA8451 is not set
 # CONFIG_THERMAL is not set
 CONFIG_WATCHDOG=y
 CONFIG_WATCHDOG_NOWAYOUT=y
@@ -915,6 +959,7 @@ CONFIG_WATCHDOG_NOWAYOUT=y
 #
 # Watchdog Device Drivers
 #
+# CONFIG_DA9052_WATCHDOG is not set
 # CONFIG_SOFT_WATCHDOG is not set
 CONFIG_MXC_WATCHDOG=y
 # CONFIG_MAX63XX_WATCHDOG is not set
@@ -926,7 +971,7 @@ CONFIG_SSB_POSSIBLE=y
 #
 # CONFIG_SSB is not set
 CONFIG_MFD_SUPPORT=y
-# CONFIG_MFD_CORE is not set
+CONFIG_MFD_CORE=y
 # CONFIG_MFD_88PM860X is not set
 # CONFIG_MFD_SM501 is not set
 # CONFIG_MFD_ASIC3 is not set
@@ -952,6 +997,7 @@ CONFIG_MFD_LTC3589=y
 CONFIG_MFD_LTC3589_I2C=y
 # CONFIG_MFD_PCF50633 is not set
 # CONFIG_MFD_MC13783 is not set
+CONFIG_PMIC_DA9052=y
 # CONFIG_ABX500_CORE is not set
 # CONFIG_EZX_PCAP is not set
 # CONFIG_AB8500_CORE is not set
@@ -972,6 +1018,7 @@ CONFIG_REGULATOR=y
 CONFIG_REGULATOR_MC13892=y
 CONFIG_REGULATOR_LTC3589=y
 CONFIG_REGULATOR_MAX17135=y
+CONFIG_REGULATOR_DA9052=y
 CONFIG_MEDIA_SUPPORT=y
 
 #
@@ -1110,6 +1157,8 @@ CONFIG_VIDEO_MXC_IPU_CAMERA=y
 # CONFIG_MXC_CAMERA_MICRON111 is not set
 # CONFIG_MXC_CAMERA_OV2640 is not set
 CONFIG_MXC_CAMERA_OV3640=y
+# CONFIG_MXC_CAMERA_OV5640 is not set
+# CONFIG_MXC_CAMERA_OV5642 is not set
 # CONFIG_MXC_TVIN_ADV7180 is not set
 CONFIG_MXC_IPU_PRP_VF_SDC=y
 CONFIG_MXC_IPU_PRP_ENC=y
@@ -1174,6 +1223,7 @@ CONFIG_FB_MXC_SYNC_PANEL=y
 # CONFIG_FB_MXC_EINK_PANEL is not set
 # CONFIG_FB_MXC_ELCDIF_FB is not set
 # CONFIG_FB_S1D13XXX is not set
+# CONFIG_FB_TMIO is not set
 # CONFIG_FB_VIRTUAL is not set
 # CONFIG_FB_METRONOME is not set
 # CONFIG_FB_MB862XX is not set
@@ -1182,6 +1232,7 @@ CONFIG_BACKLIGHT_LCD_SUPPORT=y
 # CONFIG_LCD_CLASS_DEVICE is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
 CONFIG_BACKLIGHT_GENERIC=y
+# CONFIG_BACKLIGHT_DA9052 is not set
 CONFIG_BACKLIGHT_PWM=y
 # CONFIG_BACKLIGHT_ADP8860 is not set
 CONFIG_BACKLIGHT_MXC_MC13892=y
@@ -1354,6 +1405,7 @@ CONFIG_RTC_INTF_DEV_UIE_EMUL=y
 # Platform RTC drivers
 #
 # CONFIG_RTC_DRV_CMOS is not set
+# CONFIG_RTC_DRV_DA9052 is not set
 # CONFIG_RTC_DRV_DS1286 is not set
 # CONFIG_RTC_DRV_DS1511 is not set
 # CONFIG_RTC_DRV_DS1553 is not set
@@ -1464,6 +1516,7 @@ CONFIG_MXC_MC13892_POWER=y
 #
 # MXC Asynchronous Sample Rate Converter support
 #
+# CONFIG_MXC_ASRC is not set
 
 #
 # MXC Bluetooth support
-- 
1.5.4.4

